incubator/portal/templates/backend.ingress.yaml

```diff
@@ -1,30 +1,39 @@
 {{- if .Values.ingress.enabled -}}
 
 {{- $root := . -}}
-{{- $servicePort := .Values.proxy.service.externalPort -}}
-{{- $serviceName := include "portal.proxy.fullname" . -}}
+{{- $serviceName := include "common.fullname" . -}}
 
 {{- $basehost := .Values.config.basehost -}}
 
 {{ range $name, $value := .Values.backends }}
 {{ if not $value.external }}
+
+{{ if $value.externalName }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.fullname" $root }}-{{ $name }}
+  labels:
+{{ include "common.labels.standard" $root | indent 4 }}
+spec:
+  type: ExternalName
+  externalName: {{ $value.externalName | quote }}
+{{ end }}
 ---
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ include "portal.proxy.fullname" $root }}-{{ $name }}
+  name: {{ include "common.fullname" $root }}-{{ $name }}
   annotations:
     kubernetes.io/tls-acme: {{ $root.Values.ingress.tls.enabled | quote }}
-    ingress.kubernetes.io/auth-signin: {{ template "portal.auth-signin" $root }}
-    ingress.kubernetes.io/auth-url: {{ template "portal.auth-url" $root }}
-    ingress.kubernetes.io/auth-request-redirect: {{ template "portal.auth-request-redirect" $root }}
     nginx.ingress.kubernetes.io/auth-signin: {{ template "portal.auth-signin" $root }}
     nginx.ingress.kubernetes.io/auth-url: {{ template "portal.auth-url" $root }}
     nginx.ingress.kubernetes.io/auth-request-redirect: {{ template "portal.auth-request-redirect" $root }}
-
-{{- if $root.Values.ingress.dns.enabled }}
-    domainName: {{ printf "%s.%s" $name $basehost  | quote }}
-{{- end }}
+{{ if $value.tls }}
+    nginx.ingress.kubernetes.io/secure-backends: "true"
+    nginx.ingress.kubernetes.io/server-snippet: "proxy_ssl_verify off;"
```

Comment:
It would be nice to have verify as its own flag. You may want TLS AND verification of certs